particle effect hits

JTAdams6501 opened the pull request:
ive addedd my base hit effects if you want a more variation and change let me know as of right now we have a blue with as a special arrow hit and white as a normal arrow hit

Colby626 committed:
Merge pull request #11 from EricWRogers/luke-branch

Finished Pressure Plate (needs art pass, code functional however), and Menu

CarsonBurnham committed:
Finalized Puzzle logic

Commit:
Finished level 1 whitebox

CarsonBurnham committed:
began work on final puzzle logic

CarsonBurnham committed:
began work on revised puzzle completion logic

Aiden-Thacker committed:
Started The Starting/Main Menu Area

LiamYork9 committed:
Merge pull request #12 from EricWRogers/christian-branch

Finished whitebox for puzzle 1

Colby626 committed:
Merge branch 'main' into colby-branch

Colby626 committed:
Progress on speedrun timer

CheckThePockets committed:
Models added

Colby626 committed:
Merge pull request #13 from EricWRogers/art3d-branch

Models added

CarsonBurnham committed:
Merge branch 'carson_branch' of https://github.com/EricWRogers/vr_workshop_2024 into carson_branch

LiamYork9 committed:
Merge remote-tracking branch 'origin/main' into liam_level_design_branch

CarsonBurnham committed:
Updated target logic in place, just needs revising

Commit:
Progress on timer

LiamYork9 committed:
Added a test scene

Colby626 committed:
Added a speedrun leaderboard

Colby626 committed:
Merge branch 'main' into colby-branch

ldroberts02 committed:
Adjusted the menu canvas to only rotate and move alongside camera on Y

Aiden-Thacker committed:
Merge branch 'main' into aiden-branch

LiamYork9 committed:
Started to fill the levels with the Props

Aiden-Thacker committed:
Redesigned the Starting Area Not finished yet

CarsonBurnham committed:
Finally completed updated puzzle logic, now functions as intended

CarsonBurnham committed:
Merge remote-tracking branch 'origin/main' into carson_branch

Colby626 committed:
Merge pull request #15 from EricWRogers/colby-branch

Colby branch

Commit:
Saved Colton's branch, this is Colton's work

Commit:
Merge branch 'main' into luke-branch

Aiden-Thacker committed:
Finished the Sky Box

Commit:
Made the menu not rotate with the player's camera

Colby626 committed:
Merge pull request #16 from EricWRogers/luke-branch

Camera/Menu Canvas Adjustment

Commit:
Merge branch 'main' into carson_branch

Aiden-Thacker committed:
Missing with the Shader Graph to get it to Work

Commit:
Cleaned up Unity error from new list being created and replaced coroutine

Colby626 committed:
Merge pull request #17 from EricWRogers/carson_branch

3 Target Timer Puzzle

Commit:
Merge branch 'main' into colby-branch

CheckThePockets committed:
Added Bushes, Torches, and camp stuff

Colby626 committed:
Merge pull request #19 from EricWRogers/art3d-branch

Added Bushes, Torches, and camp stuff

CarsonBurnham committed:
Began work on explosive barrels

Aiden-Thacker committed:
Merge remote-tracking branch 'origin/main' into aiden-branch

Colby626 committed:
Merge branch 'main' into colby-branch

Colby626 committed:
Improved FPS controller and Arrow spawning, and added functionality for spawning something when hitting a target

Aiden-Thacker committed:
Given up on Shader Graph turned to Particle System

LiamYork9 committed:
Placed all current models into Scenes

Aiden-Thacker committed:
Almost done with the Starter Area

Colby626 committed:
Merge pull request #20 from EricWRogers/liam_level_design_branch

Model Implementation

Colby626 committed:
Merge branch 'main' into colby-branch

Aiden-Thacker committed:
Need colliders than it will be done

Colby626 committed:
Added new VR bow functionality of spawning and despawning

Colby626 committed:
Hooked up everything but the start and end to beat the game with WASD

Colby626 committed:
Bow now spawns in the player's hand and despawns with press and release of grip

Colby626 committed:
Arrows spawn on right grip

Colby626 committed:
Made the bow not push the player

ldroberts02 committed:
"white" box end screen

ldroberts02 committed:
Merge branch 'luke-branch' of https://github.com/EricWRogers/vr_workshop_2024 into luke-branch

Aiden-Thacker committed:
Added Colliders and Finished the Starter Area

You will need to change the script changeLevel but other than that it works

Colby626 committed:
Merge branch 'luke-branch' into colby-branch

ldroberts02 committed:
Merge remote-tracking branch 'origin/aiden-branch' into luke-branch

Colby626 committed:
Merge pull request #21 from EricWRogers/aiden-branch

Finished the Starter Area

ldroberts02 committed:
Modified winRoom slightly for credits and book pedistal

Aiden-Thacker committed:
Merge remote-tracking branch 'origin/colby-branch' into aiden-branch

Aiden-Thacker committed:
Basically a Build

Late Night Programming Dont Ask What I have Made Ok

Colby626 committed:
Arrow fires from VR bow now

Colby626 committed:
Setup a win material for the final trial and prevented hands from stealing from each other

Colby626 committed:
Quiver spawns arrow

Commit:
Merge branch 'main' into colby-branch

Commit:
Merge aiden's branch

JTAdams65 committed:
Merge remote-tracking branch 'origin/main' into joshua_branch

Commit:
Everything works now :)

Colby626 committed:
Merge pull request #22 from EricWRogers/joshua_branch

Joshua branch

Commit:
Reload zone, testing quiver

Commit:
Merge main

Commit:
Updated bow trail

Colby626 committed:
Fixed console error from ShowCredits and made quiver work as intended

Colby626 committed:
Fixed build errors

Aiden-Thacker committed:
Merge pull request #18 from EricWRogers/colby-branch

Saved Colton's branch, this is Colton's work

Aiden-Thacker committed:
Merge remote-tracking branch 'origin/main' into aiden-branch

Aiden-Thacker committed:
Added Fire to Wall Torch and Campfire. Also did Turn table

Colby626 committed:
Merge main, overwritting this branch's changes

Colby626 committed:
Merge pull request #23 from EricWRogers/luke-branch

White Boxed End Area

LiamYork9 committed:
Merge branch 'main' into liam_level_design_branch

Commit:
Auto stash before merge of "main" and "origin/main"

Commit:
Merge branch 'main' into christian-branch

Aiden-Thacker committed:
Worked on the Fire of the Torches and Fire Pit

Aiden-Thacker committed:
Added Fire to the Braziers

LiamYork9 committed:
Merge pull request #24 from EricWRogers/aiden-branch

Finished the Turn table Adding Fire to Torches, Camp Fire, and Braziers

LiamYork9 committed:
Merge branch 'main' into liam_level_design_branch

CheckThePockets committed:
Added some 3d stuff and textures

CheckThePockets committed:
Merge branch 'main' into art3d-branch

CarsonBurnham committed:
Working on specific object deletion

Commit:
Added Sound Effects

Wood, Stone, and Metal impact. Footsteps for stone and dirt. Pulling back bowstring. Arrow whish.

Commit:
Merged main into branch

Commit:
Merged main again

Colby626 committed:
Merge pull request #25 from EricWRogers/art3d-branch

Added some 3d stuff and textures

Colby626 committed:
Merge pull request #26 from EricWRogers/christian-branch

Christian branch

LiamYork9 committed:
moved 2nd puzzle forward

Colby626 committed:
Made black circle in VR go away in start scene, deleted exit game script

Aiden-Thacker committed:
Merge pull request #27 from EricWRogers/colby-branch

Made black circle in VR go away in start scene, deleted exit game script

LiamYork9 committed:
Merge branch 'main' into liam_level_design_branch

LiamYork9 committed:
Started on the trial one rework

Aiden-Thacker committed:
Merge branch 'main' into aiden-branch

CarsonBurnham committed:
Finished Explosive Barrel

CarsonBurnham committed:
Merge branch 'main' into carson_branch

Aiden-Thacker committed:
Adding to the Stater Area

CarsonBurnham committed:
Began work on star twinkling

LiamYork9 committed:
started changing the build scene

Colby626 committed:
Made the quiver act much better than before

Colby626 committed:
Updated a position in playmode last commit, this one has good quiver

Colby626 committed:
Even better quiver

Colby626 committed:
Applied quiver change to prefab

LiamYork9 committed:
Merge pull request #28 from EricWRogers/colby-branch

Colby branch

LiamYork9 committed:
Merge branch 'main' into liam_level_design_branch

LiamYork9 committed:
Did work on tailing and adding more props

CheckThePockets committed:
Merge branch 'art3d-branch' of https://github.com/EricWRogers/vr_workshop_2024 into art3d-branch

CheckThePockets committed:
new bow and pallet

CheckThePockets committed:
Merge branch 'main' into art3d-branch

Colby626 committed:
Merge pull request #29 from EricWRogers/art3d-branch

Art3d branch

CheckThePockets committed:
more 3d stuff and textures

CheckThePockets committed:
Merge branch 'main' into art3d-branch

Colby626 committed:
Merge pull request #30 from EricWRogers/art3d-branch

Art3d branch

Commit:
Added sounds

LiamYork9 committed:
Added more models, and texturing as well as fixing a scaling problom

LiamYork9 committed:
Merge branch 'main' into liam_level_design_branch

LiamYork9 committed:
redoing

CarsonBurnham committed:
Twinkling stars need polish, but otherwise finished

Aiden-Thacker committed:
Working on Low Poly Water

CarsonBurnham committed:
Finished twinkling stars, prefab in my folder

CarsonBurnham committed:
minor change to stars

LiamYork9 committed:
Redid the starting area

Commit:
Added Arrow Script to my scene

Aiden-Thacker committed:
Worked on Water Plane and Starter Area Layout

Colby626 committed:
Merge pull request #31 from EricWRogers/liam_level_design_branch

Liam level design branch

Colby626 committed:
Implemented a speedrun timer

LiamYork9 committed:
Merge pull request #32 from EricWRogers/colby-branch

Implemented a speedrun timer

Aiden-Thacker committed:
Merge branch 'main' into aiden-branch

Aiden-Thacker committed:
Added New Models into the Starter Scene

I will need to work on colliders soon

LiamYork9 committed:
Merge branch 'main' into liam_level_design_branch

Aiden-Thacker committed:
Created Mountains for the Starter Scene

CheckThePockets committed:
New Arrows and vines

Colby626 committed:
Merge pull request #34 from EricWRogers/art3d-branch

New Arrows and vines

CarsonBurnham committed:
Began work on parchment popup for notes

LiamYork9 committed:
New Scene

Font Added

Aiden-Thacker committed:
Merge pull request #35 from EricWRogers/carson_branch

Stars and work on Parchment

Commit:
WASDPlayer in my scene, sound when fire arrow works

Sound when fire arrow worlks and WASDPlayer in my scene

LiamYork9 committed:
Trees

Aiden-Thacker committed:
Added a Collider and Trees

Commit:
Started integrating liam's longer scene

Colby626 committed:
Merge branch 'main' into colby-branch

Colby626 committed:
Removed duplicate arrow tag and added leaderboard wiping and currentRunHighscore

Colby626 committed:
Added input field for name inputs on highscores

Aiden-Thacker committed:
Added More Trees and Colliders

Aiden-Thacker committed:
Merge branch 'main' into aiden-branch

Aiden-Thacker committed:
Added Moon and Carson's Stars

LiamYork9 committed:
Added Font

LiamYork9 committed:
Merge branch 'main' into liam_level_design_branch

Commit:
Initalize top5 in LoadLeaderboard

Commit:
Made names save properly

Commit:
Prefabbed the input field for the name

Commit:
Footsteps play while walkie, but there's no limit on amount of sounds

Step sounds work, too many step sounds per step

LiamYork9 committed:
Built the second puzzle for the second area

Aiden-Thacker committed:
Merge pull request #36 from EricWRogers/liam_level_design_branch

Liam level design branch

LiamYork9 committed:
Built out the 2nd Trial Puzzles

Aiden-Thacker committed:
Started Target Ranges

Commit:
Merge branch 'main' into colby-branch

CheckThePockets committed:
new arrows and bow

Colby626 committed:
Merge pull request #37 from EricWRogers/art3d-branch

new arrows and bow

Commit:
Merge branch 'main' into colby-branch

Aiden-Thacker committed:
You Now Can Go across the River

LiamYork9 committed:
I have filled in the 2nd Trail area.

Fully white boxed out the first two puzzles last one just needs walls

LiamYork9 committed:
Merge branch 'main' into liam_level_design_branch

LiamYork9 committed:
Added scroll and end gate

Colby626 committed:
Merge pull request #38 from EricWRogers/liam_level_design_branch

Liam level design branch

Commit:
Made the new changes to the first trial be interactable and new color changes on the target

Commit:
Footstep audio works

Alternates between step 1 and 2

Colby626 committed:
Started rigging the bow

Aiden-Thacker committed:
Change Old Script to ToNewScene

Aiden-Thacker committed:
Merge branch 'main' into aiden-branch

Aiden-Thacker committed:
Fixed Bridge Error and Use Goes to Liam's Build Scene

LiamYork9 committed:
Merge branch 'main' into liam_level_design_branch

LiamYork9 committed:
I have fully whiteboxed the level, added a basic reloud zone and placed the WASD Player

Aiden-Thacker committed:
Merge pull request #40 from EricWRogers/liam_level_design_branch

Whiteboxed Area 2

LiamYork9 committed:
Merge pull request #39 from EricWRogers/aiden-branch

Starter Scene (Finished For Now)

Colby626 committed:
Merged aiden's branch into colby's branch

Colby626 committed:
Merge branch 'main' into colby-branch

Colby626 committed:
First scene done

Colby626 committed:
Changed project settings to allow respawn points

Colby626 committed:
Progress on fire trial

Colby626 committed:
Basically a build in WASD and working on VR now

Colby626 committed:
Fixed bow and quiver

Colby626 committed:
New build for VR

Aiden-Thacker committed:
Merge pull request #41 from EricWRogers/colby-branch

Colby branch

LiamYork9 committed:
Merge branch 'main' into liam_level_design_branch

Commit:
Merge branch 'main' into christian-branch

Commit:
arrow is bigger and with continuous detection

Aiden-Thacker committed:
Merge pull request #42 from EricWRogers/christian-branch

Christian branch

LiamYork9 committed:
Resized Trial_2 and added a color to the Lanterns

LiamYork9 committed:
Merge branch 'main' into liam_level_design_branch

Aiden-Thacker committed:
Merge pull request #43 from EricWRogers/liam_level_design_branch

Resizing of the 2nd Trial

Colby626 committed:
Progress on a rigged bow

Colby626 committed:
Merge branch 'colby-branch' of https://github.com/EricWRogers/vr_workshop_2024 into colby-branch

Colby626 committed:
Merge branch 'main' into colby-branch

Colby626 reviewed (changes requested):
Update the hit effect to what you have from your home computer

Aiden-Thacker committed:
Merge branch 'main' into aiden-branch

Commit:
Prefab for AudioManagerObject

CarsonBurnham committed:
Merge branch 'christian-branch' into note-interaction

Aiden-Thacker committed:
Change the Statue To have a Quiver

CheckThePockets committed:
Textures and gators

Colby626 committed:
Merge pull request #45 from EricWRogers/art3d-branch

Textures and gators

LiamYork9 committed:
I have changed the lighting in the 2 sets of trials

I have also added black pits under the areas that needed them

LiamYork9 committed:
Merge branch 'main' into liam_level_design_branch

Aiden-Thacker committed:
Exploring the Arrow Prefab to fix the Trailing Effect

Commit:
Working on torch sound

Commit:
Added WreckBow premade bow and arrow VR project

Commit:
Merge branch 'main' into colby-branch

CarsonBurnham committed:
Note interaction almost done

Aiden-Thacker committed:
Merge pull request #46 from EricWRogers/liam_level_design_branch

Liam level design branch

Aiden-Thacker committed:
I missed with the Lights, Explosion VFX, and Arrow

Made the Lights More Orange on the particle system did not touch light source

Aiden-Thacker committed:
The Player can Exit Play Mode in Win Scene

Aiden-Thacker committed:
Working on the Arrow Trail some more

Colby626 committed:
Merge branch 'main' into colby-branch

Colby626 committed:
Progress on rigged bow

Colby626 committed:
Float to track how far back the bow is pulled

Colby626 committed:
Bow string works

Colby626 committed:
Quiver on back

Colby626 committed:
Bow spawns properly

Aiden-Thacker committed:
Worked on Arrow Trail and Reset some scripts I touched

Aiden-Thacker committed:
Merge remote-tracking branch 'origin/christian-branch' into aiden-branch

Aiden-Thacker committed:
Did some Null Reference checking

LiamYork9 committed:
Added textures to the walls, fixed up the lighting in the levels

I made the tunnel shorter and also added more time to the last puzzle of the first trial

Aiden-Thacker committed:
Merge pull request #47 from EricWRogers/liam_level_design_branch

Added textures to the walls, fixed up the lighting in the levels

Aiden-Thacker committed:
Worked on Colliders between the Arrow and Target

Aiden-Thacker committed:
Worked on the Target Prefab in the Starter Area

Aiden-Thacker committed:
Merge branch 'main' into aiden-branch

Aiden-Thacker committed:
Added Audio Managers and mess with Particle Systems on the Lantern

Colby626 committed:
Fixed audiomanager bug between scenes

Colby626 committed:
Garbage collection on scene load for the audiomanager

Colby626 committed:
Rigged bow can nock arrows and fire them

Colby626 committed:
Back of the arrow is now nocked properly

Colby626 committed:
Arrows ignore the bow for physics

Colby626 committed:
Merged aiden's into mine, accepted mine

CarsonBurnham committed:
Merge remote-tracking branch 'origin/main' into note-interaction

CarsonBurnham committed:
Bugfixing Note Interaction

CarsonBurnham committed:
I fixed it

Note Interaction now works

LiamYork9 committed:
Merge pull request #50 from EricWRogers/note-interaction

Note interaction

Colby626 committed:
Rigged bow build

Colby626 committed:
Merge branch 'colby-branch'

Colby626 committed:
Removed one copy of the Arrow tag from the tagManager (not showing in Git) and changed the Arrow prefab to use the other Arrow tag

Colby626 committed:
Now the tagmanager shows up as changed but not before?? Might remove the ghost arrow tag

Colby626 committed:
Fixed an error in the console when the player's hand initialized in the quiver grab area

Colby626 committed:
Removed unnecessary variables

Colby626 committed:
Removed the arrow pushing the player bug

Aiden-Thacker committed:
Merge pull request #51 from EricWRogers/colby-branch

Colby branch
